cb-sol-cli/cmd/admin.js

```diff
@@ -0,0 +1,115 @@
+const ethers = require('ethers');
+const constants = require('../constants');
+
+const {Command} = require('commander');
+const {setupParentArgs, waitForTx, log} = require("./utils")
+
+const isRelayerCmd = new Command("is-relayer")
+    .description("Check if address is relayer")
+    .option('--relayer <value>', 'Address to check', constants.relayerAddresses[0])
+    .option('--bridge <address>', 'Bridge contract address', constants.BRIDGE_ADDRESS)
+    .action(async function (args) {
+            await setupParentArgs(args, args.parent.parent)
+            const bridgeInstance = new ethers.Contract(args.bridge, constants.ContractABIs.Bridge.abi, args.wallet);
+
+            let res = await bridgeInstance.isRelayer(args.relayer)
+            console.log(`[${args._name}] Address ${args.relayer} ${res ? "is" : "is not"} a relayer.`)
+    })
+
+const addRelayerCmd = new Command("add-relayer")
+    .description("Add a relayer")
+    .option('--relayer <address>', 'Address of relayer', constants.relayerAddresses[0])
+    .option('--bridge <address>', 'Bridge contract address', constants.BRIDGE_ADDRESS)
+    .action(async function (args) {
+        await setupParentArgs(args, args.parent.parent)
+        const bridgeInstance = new ethers.Contract(args.bridge, constants.ContractABIs.Bridge.abi, args.wallet);
+        log(args, `Adding ${args.relayer} as a relayer.`)
+        let tx = await bridgeInstance.adminAddRelayer(args.relayer)
+        await waitForTx(args.provider, tx.hash)
+    })
+
+const removeRelayerCmd = new Command("remove-relayer")
+    .description("Remove a relayer")
+    .option('--relayer <address>', 'Address of relayer', constants.relayerAddresses[0])
+    .option('--bridge <address>', 'Bridge contract address', constants.BRIDGE_ADDRESS)
+    .action(async function (args) {
+        await setupParentArgs(args, args.parent.parent)
+        const bridgeInstance = new ethers.Contract(args.bridge, constants.ContractABIs.Bridge.abi, args.wallet);
+        log(args, `Removing relayer ${args.relayer}.`)
+        let tx = await bridgeInstance.adminRemoveRelayer(args.relayer)
+        await waitForTx(args.provider, tx.hash)
+    })
+
+const setThresholdCmd = new Command("set-threshold")
```

Comment:
```suggestion
const setThresholdCmd = new Command("set-relayer-threshold")
```
to differentiate from the proposal expiration threshold if we choose to add it/other threshold related functions in the future